jujugui/static/gui/src/app/app.js

```diff
@@ -491,7 +491,7 @@ YUI.add('juju-gui', function(Y) {
       let password = null;
       let macaroons = null;
       let credentials = this.user.controller;
-      if (credentials.areAvailable) {
+      if (credentials && credentials.areAvailable) {
```

Comment:
@frankban As fars as the further errors you get, I think we'll continue to get errors as more stuff is not set up correctly, I guess we just need to get it working to the point where that notification can be displayed...